test/com/dataengi/crm/profiles/daos/ProfilesDAOSpec.scala

```diff
@@ -1,28 +1,47 @@
 package com.dataengi.crm.profiles.daos
 
-import play.api.test.PlaySpecification
-import com.dataengi.crm.profiles.context.ProfilesContext
 import com.dataengi.crm.common.context.types._
+import com.dataengi.crm.common.extensions.arbitrary._
 import com.dataengi.crm.common.extensions.awaits._
-import com.dataengi.crm.common.extensions.logging._
-import org.scalacheck.{Arbitrary, Gen}
+import com.dataengi.crm.profiles.context.ProfilesContext
+import com.dataengi.crm.profiles.models.Profile
 import org.specs2.runner.SpecificationsFinder
+import play.api.test.PlaySpecification
 
-/**
-  * Created by nk91 on 31.01.17.
-  */
-class ProfilesDAOSpec extends PlaySpecification with ProfilesContext with SpecificationsFinder {
+class ProfilesDAOSpec extends PlaySpecification with ProfilesContext {
 
   "ProfilesDAOSpecification" should {
 
+    "get option profile by id" in {
+
+      val getProfileResult = profilesDAO.getOption(24254625625L).await()
```

Comment:
try to avoid the magic number
extract it to value `NonExistId` or take max Int value